src/components/ConversationWindow.js

```diff
@@ -663,7 +671,8 @@ ConversationWindow.propTypes = {
 ConversationWindow.defaultProps = {
   Message: MessageSimple,
   showHeader: true,
-  showComposerActions: true
+  showComposerActions: true,
+  noRecordFoundError: LANGUAGE_PHRASES.NO_CONVERSATION_SELECTED 
```

Comment:
Please rename this to `noConversationFoundMessage` as `noRecordFoundError` is making sense for no conversation found as well as no messages found.

Reply:
Done!